test/sanitize_json.d

```diff
@@ -0,0 +1,231 @@
+import std.typecons : Flag, Yes, No;
+import std.array : Appender, appender;
+import std.bigint;
+import std.internal.cstring : tempCString;
+import std.string : startsWith, replace;
+import std.format : formattedWrite, format;
+import std.file : readText;
+import std.stdio;
+
+import json;
+
+__gshared File outFile;
```

Comment:
Official code should use `shared` whenever possible, though in this case neither is needed.